Editor/UI/CreatorLayout.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Diagnostics;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Innoactive.Creator.Core.Editor.UI
+{
+    public static class CreatorLayout
```

Comment:
should this be public or better move it internal? (which will also disallow us from using it in another components)

Reply:
I dont mind being it public but then a few lines of docu would be nice ;)

Reply:
For me it's okay to be public. So everyone can adapt their layout.

Reply:
If we use it in other places then this is an util and either the name or namespace feels odd.

Reply:
namespace is wrong anyway :D

Reply:
the name itself is mostly based on the same naming scheme unity uses:
GUILayout
EditorGUILayout
CreatorLayout